pandas/tests/indexes/test_interval_new.py

```diff
@@ -0,0 +1,310 @@
+from __future__ import division
```

Comment:
I suspect we will at some point need to merge `test_interval` and `test_interval_new` together.  Not sure if this PR is the right time to do that though.

Reply:
no i explicitly asked that these be kept separate for now

Reply:
Okay, gotcha.  Just wanted to check.